commonui/src/main/java/com/hyperwallet/android/ui/common/view/ActivityUtils.java

```diff
@@ -62,6 +64,11 @@ public static void showError(@NonNull final FragmentActivity fragmentActivity,
         if (fragment == null) {
             fragment = DefaultErrorDialogFragment.newInstance(errors);
         }
+        HyperwalletInsight.getInstance().trackError(fragmentActivity.getBaseContext(), "", "",
+                new HyperwalletInsight.ErrorParamsBuilder()
+                        .type(ErrorTypes.getErrorType(errors.get(0).getCode()))
+                        .description(ErrorTypes.getStackTrace())
```

Comment:
we can set the following here too:

- error code from `errors.get(0).getCode()`
- error message from `errors.get(0).getMessage()`